contrib/docker/docker-entrypoint.sh

```diff
@@ -5,6 +5,10 @@ if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "local" ]; then
     superset runserver -d
+elif [ "$SUPERSET_ENV" = "development" ]; then
+    superset runserver -d &
+    cd /home/work/incubator-superset/superset/assets
```

Comment:
Does this directory structure exist inside of the container?

Reply:
Oh yes.  See
https://github.com/apache/incubator-superset/blob/master/contrib/docker/Dockerfile#L11
https://github.com/apache/incubator-superset/blob/master/contrib/docker/Dockerfile#L32-L34